src/api/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:18.16.0
+
+# Create api directory
+WORKDIR /api
+
+# Install app dependencies
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
```

Comment:
```suggestion
```
Do your best to keep your comments and documentation up to date

Reply:
Took out wildcard comment